src/components/graph/Graph.tsx

```diff
@@ -110,23 +111,41 @@ function Graph() {
 	);
 
 	const panelsRange: Array<PanelRange> = React.useMemo(() => {
-		return [
-			{
-				type: 'events-panel',
-				range: activeWorkspace.eventsStore.panelRange,
-				setRange: activeWorkspace.eventsStore.onRangeChange,
-			},
-			{
-				type: 'messages-panel',
-				range: activeWorkspace.messagesStore.panelRange,
-				setRange: activeWorkspace.messagesStore.onRangeChange,
-			},
-		];
-	}, [activeWorkspace.eventsStore.panelRange, activeWorkspace.messagesStore.panelRange]);
+		return isWorkspaceStore(activeWorkspace)
```

Comment:
I think a better way to solve this would be to have an intermediate component that returns just graph background if active workspace is SearchWorkspace and return Graph component otherwise